### PR TITLE
changing download instructions

### PR DESCRIPTION
**Pull request by cmastr**
Signed-off-by: Casie Oxford <coxford@d2iq.com>

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4297.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.2/download/index.md [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+```
+
+You will then see the following output:
+
+```sh
+dkp binary is placed in the local directory, to run:
+./dkp --help
+```
+
+You will now see two binaries: `dkp` and `kommander` in your working directory. Follow the [Kommander installation instructions](../install/networked) using these binaries, and then [add your license](../licensing/add/) to Kommander. If you have problems downloading or installing Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```
Comment (mmcelroy91) [inline pages/dkp/konvoy/2.2/download/index.md:42]:
```suggestion
You will now see the `dkp` binary in your working directory.. Follow the [Kommander installation instructions](../install/networked) using these binaries, and then [add your license](../licensing/add/) to Kommander. If you have problems downloading or installing Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```

Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/download/index.md:42]: Made this change per Dimitri in Slack documentation channel.

File: pages/dkp/kommander/2.2/download/index.md [hunk 2]
```diff
@@ -31,9 +35,8 @@ docker run -it --rm -u $(id -u):$(id -g) -v $(pwd):/dkp $CONTAINER_IMAGES
 You will then see the following output:
 
 ```sh
-dkp and kommander binaries placed in the local directory, to run:
+dkp binary is placed in the local directory, to run:
 ./dkp --help
-./kommander --help
 ```
 
 You will now see two binaries: `dkp` and `kommander` in your working directory. Follow the [Kommander installation instructions](../install/networked) using these binaries, and then [add your license](../licensing/add/) to Kommander. If you have problems downloading or installing Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.2/download/index.md:42]:
```suggestion
You will now see the dkp binary in your working directory. Follow the [Kommander installation instructions](../install/networked) using these binaries, and then [add your license](../licensing/add/) to Kommander. If you have problems downloading or installing Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```

Comment (mmcelroy91) [reply pages/dkp/kommander/2.2/download/index.md]: Here it is without period extra period.

File: pages/dkp/kommander/2.2/download/index.md [hunk 3]
```diff
@@ -1,22 +1,26 @@
...
-To download a new version of Kommander, you have 2 options:
+To download a new version of DKP, you have 2 options:
 
 ## Download from the support website
 
-[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download DKP[/button]
 
-<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product.</p>
+<p class="message--note"><strong>NOTE: </strong> In DKP 2.2 the Konvoy and Kommander binaries have been merged into a single binary, which you can find by selecting the DKP button above.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.2/download/index.md:19]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>In DKP 2.2, the Konvoy and Kommander binaries have been merged into a single binary, which you can find by selecting the DKP button above.
```

File: pages/dkp/konvoy/2.2/download/index.md [hunk 4]
```diff
@@ -1,19 +1,42 @@
...
+To download a new version of DKP, you have 2 options:
 
-[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+## Download from the support website
 
-<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product.
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download DKP[/button]
 
-If you have problems downloading Konvoy, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.</p>
+<p class="message--note"><strong>NOTE: </strong> In DKP 2.2 the Konvoy and Kommander binaries have been merged into a single binary, which you can find by selecting the DKP button above.
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.2/download/index.md:19]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>In DKP 2.2, the Konvoy and Kommander binaries have been merged into a single binary, which you can find by selecting the Download DKP button above.
```

File: pages/dkp/konvoy/2.2/download/index.md [hunk 5]
```diff
@@ -1,19 +1,42 @@
 ---
 layout: layout.pug
 navigationTitle: Download
-title: Download Konvoy
-excerpt: Learn how to download Konvoy
+title: Download DKP
+menuWeight: 2
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.2/download/index.md:5]: Don't know if this matters, but this pushes it above the Release notes in Konvoy

**Review by msdolbey — APPROVED** [review]
👍

**Pull request merged**